addons/fastroping/functions/fnc_fastRopeLocalPFH.sqf

```diff
@@ -31,10 +31,19 @@ if (animationState _unit != "ACE_FastRoping") exitWith {
     [_unit, "ACE_FastRoping", 2] call EFUNC(common,doAnimation);
 };
 
+[_unit] spawn {
```

Comment:
No need for waitUntilAndExecute. A succesion of waitAndExecute, that each check if the unit is still fastroping should be enough